lib/galaxy/tools/deps/container_classes.py

```diff
@@ -263,12 +273,17 @@ def containerize_command(self, command):
             volumes_from=volumes_from,
             env_directives=env_directives,
             working_directory=working_directory,
-            net=self.prop("net", "none"),  # By default, docker instance has networking disabled
+            net=self.prop("net", None),  # By default, docker instance has networking disabled
```

Comment:
This change seems wrong? Won't this disable disabling network access by default or was there a bug here already?

Reply:
@bgruening was fixing a bug?

Reply:
I think this fixed a bug in my testing. The former yielded to `--net=None` I think and not to ` ` which was expected.